deploy.sh

```diff
@@ -6,14 +6,9 @@ source .env
 
 DEPLOY_ENV=$1
 
-DJANGO_SETTINGS_MODULE=teamware.settings.deployment
-export DJANGO_SETTINGS_MODULE
-
 case $DEPLOY_ENV in
 
   production|prod)
-    DJANGO_ALLOWED_HOSTS=$TEAMWARE_HOST_URL_PRODUCTION
-    export DJANGO_ALLOWED_HOSTS
```

Comment:
The inherited process environment overrides `.env` for `docker-compose`, so we can declare `DJANGO_ALLOWED_HOSTS` directly in the `.env` but still override it if we want to for staging/test.  This then means that `./deploy.sh prod` is directly equivalent to a simple `docker compose up -d`